activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-core/src/main/java/org/activiti/cloud/modeling/core/error/ModelNameInvalidException.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.activiti.cloud.modeling.core.error;
+
+public class ModelNameInvalidException extends ModelingException {
```

Comment:
Can we rename this (and related prod/test methods) to `InvalidModelNameException`?

Reply:
Done